credentials.json.enc

```diff
@@ -0,0 +1,49 @@
+U2FsdGVkX1/vVm/dOEg1DCACYbdOcL+ey6+64A+DZGZVgF8Z/3skK6rpPocu6GOA
```

Comment:
Is this credentials file being used by anything? If it's from google-cloud-python it may just be cruft

Reply:
Removed!